http/css/style.css

```diff
@@ -65,6 +65,11 @@ p { max-width: 800px; }
 .hCell-sep       { width: 0.3em; }
 .hCell-driver    { min-width: 5.1em; }
 
+.permalink { margin-left: 0.3em; display: none; }
+.permalink:link { text-decoration: none; color: #DDDDDD; }
+.permalink:visited { text-decoration: none; color: #DDDDDD; }
+:hover > .permalink { display: inline; }
```

Comment:
I tried with that first, and it also worked, but I haven't found any reference to `initial` value in the standard: https://developer.mozilla.org/fr/docs/Web/CSS/display, so I'm not very keen to use it :neutral_face: